### ISSUE TITLE
Fix wave death on slopes with d blocks

### ISSUE THREAD

**Issue by Nicko2102** [absoIute/Mega-Hack-Pro-Future#392]
When in practice mode, if you place a checkpoint on a slope with a D block on it and respawn there, you die. Can you add a fix for this (or let me know if this is a duplicate) please?

**Comment by Nicko2102** [comment]
For example: practicing the first wave in nhelv is impossible, because its super hard to place a checkpoint after letting go of the mouse and before hitting the slope under you

**Comment by Brittank88** [comment]
Yeah, this is a major annoyance and makes playtesting or practising certain levels a PITA. Would love to see this fix implemented.
